engine/src/build/config/ios/BUILD.gn

```diff
@@ -6,6 +6,58 @@ config("sdk") {
   cflags_cc = [ "-stdlib=libc++" ]
 }
 
+config("compiler") {
+  # CPU architecture.
+  if (current_cpu == "x64") {
+    _triplet_cpu = "x86_64"
+  } else if (current_cpu == "x86") {
+    _triplet_cpu = "i386"
+  } else if (current_cpu == "arm" || current_cpu == "armv7") {
+    _triplet_cpu = "armv7"
```

Comment:
These cases shouldn't happen, I'm pretty sure.

Reply:
Good point; I've moved these to a comment in the assert(false) block below.